ToDoLite/DetailViewController.m

```diff
@@ -226,6 +246,18 @@ - (UITableViewCell *)couchTableSource:(CBLUITableSource *)source cellForRowAtInd
                                                     forIndexPath:indexPath];
     CBLQueryRow *row = [source rowAtIndex:indexPath.row];
     Task *task = [Task modelForDocument:row.document];
+
+    int64_t delay = 1.5; // In seconds
```

Comment:
After https://github.com/couchbase/couchbase-lite-ios/issues/1132 is resolved, will this still be needed?